receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/accumulator.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package awsecscontainermetrics
+
+import (
+	"time"
+
+	metricspb "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1"
+	resourcepb "github.com/census-instrumentation/opencensus-proto/gen-go/resource/v1"
```

Comment:
@asuresh4 @bogdandrutu Are metrics receivers still using opencensus proto?

Reply:
No, we changed most of the core to use the otlp and internal structs. Completely recommend for new components to avoid oc.

Reply:
@hossain-rayhan you need to start using `pdata.Metrics`

Reply:
Hi @bogdandrutu , before sending the data to next consumer I am using `internaldata.OCToMetrics(md)` to convert our metrics to `pdata.Metrics`. Wondering, isn't that enough like other receivers in the repo or we should strictly get rid of it now?

Reply:
That is a temporary solution to make progress and not have to change all components once. And decided to use that for some old components that we did not have time to chnage

Reply:
@hossain-rayhan Yeah so basically you should only be converting at the last moment when passing down, but here in this sort of receiver-specific logic we want to be using `pdata`, the OTel format. Or we just have to rewrite it right away. We're also having data-model issues because of using the old format (Resource type for example) and we want to make sure the model is right

Reply:
Thanks @bogdandrutu and @anuraaga. I understand we need to use `pdata` eventually. I was planning to move forward with this to meet our internal deadline. We can send a different PR after October 15th I guess. How do you guys feel about it?

Reply:
As long as you create an issue and assign to you and @anuraaga I am fine. I trust that you will fix it. I will let @anuraaga make the final call here.

Reply:
Issue created: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/1122